src/overlays/actors/ovl_En_Tp/z_en_tp.c

```diff
@@ -160,7 +160,8 @@ void EnTp_Init(Actor* thisx, PlayState* play2) {
             next = (EnTp*)Actor_Spawn(&play->actorCtx, play, ACTOR_EN_TP, this->actor.world.pos.x,
                                       this->actor.world.pos.y, this->actor.world.pos.z, 0, 0, 0, 0 * i);
 
-            if (0 * i) {} // Very fake, but needed to get the s registers right
+            if ((0 * i) != 0) {} // Very fake, but needed to get the s registers right
```

Comment:
Synray actually had a better idea for this, might be worth using instead: https://discord.com/channels/688807550715560050/688851337085190255/887885019438272523